app/controllers/buyers/applications_controller.rb

```diff
@@ -177,7 +183,11 @@ def find_cinstance
   end
 
   def find_buyer
-    @buyer = current_account.buyers.find(params[:account_id])
+    @account = current_account.buyers.find(params[:account_id])
```

Comment:
Would it be acceptable if we did
```suggestion
    account_id = params[:account_id]
    return unless account_id
    @account = current_account.buyers.find(account_id)
```
and then extended the `before_action` back to covering more actions (create, new, index)?

Reply:
I guess it would. I don't like hiding things though!